graphql_compiler/tests/test_input_data.py

```diff
@@ -1565,6 +1586,27 @@ def has_edge_degree_op_filter_with_fold():  # noqa: D103
         type_equivalence_hints=None)
 
 
+def is_null_op_filter_optional_value():  # noqa: D103
+    graphql_input = '''{
+        Animal {
+            name @output(out_name: "name")
+            net_worth @filter(op_name: "is_null")
+        }
+    }'''
+
```

Comment:
nit: follow the same whitespace pattern as the other functions unless there's good reason to break from it, this blank line and the one on line 1600 should be removed for consistency